notes/notesRouters.js

```diff
@@ -0,0 +1,111 @@
+const express = require("express");
+const notesDb = require("./notesDataModel");
+const router = express.Router();
+
+router.get("/", (req, res) => {
+  notesDb
+    .findAll()
+    .then(notes => {
+      notes.forEach(note => {
+        if (!note.tags) {
+          note.tags = [];
+        } else {
+          let tagsArr = note.tags;
+
+          console.log(tagsArr);
+          note.tags = tagsArr.split(",");
+        }
+      });
+      res.status(200).json(notes);
+    })
+    .catch(err => {
+      res.json(err);
```

Comment:
Write your own error so you know exactly where the problem occurs. Something like "database could not GET notes"